source/manual/get-started.html.md

```diff
@@ -17,24 +17,278 @@ If you're having trouble with this guide, you can ask your colleagues or the #go
+[merging]: /manual/merge-pr.html
+
+### Access remote environments
+
+Your pull request from earlier will hopefully have been merged by now. It's time to test your access to servers via SSH.
+
+> If you're not in the office right now, you'll need to be connected to the GDS office VPN for SSH access to integration.
+
+Test that it works by running:
+
+```bash
+gds govuk connect --environment integration ssh backend
+```
+
+#### Running a console
+Once you have SSH'd into a machine, you can also open a console for a particular application so you can execute commands, for example:
+
+```bash
+govuk_app_console publishing-api
+```
+
+## 5. Get AWS access
+
+To work with [govuk-aws](https://github.com/alphagov/govuk-aws) and [govuk-aws-data](https://github.com/alphagov/govuk-aws-data),
+you will require an account in AWS.
+
+### Request a GDS AWS account
+
+GDS maintains a central account for AWS access. You will need to request an account from the Technology and Operations team.
+
+👉 [Request an AWS account](https://gds-request-an-aws-account.cloudapps.digital)
+
+You'll want to click "Request user access" - NOT "Request an account". After submitting the form, you should receive an email to say your account creation is in progress, and later another email saying the work has been completed. You can then move onto step 2.
+
+### Sign in to AWS
+
+To sign in, go to [the GDS AWS Sign page](https://gds-users.signin.aws.amazon.com/console), and use the following credentials:
+
+- "Account ID or alias": `gds-users`
+- Username: your `@digital.cabinet-office.gov.uk` email address
+- Password: your password
+
+👉 [Sign in to AWS GDS account](https://gds-users.signin.aws.amazon.com/console)
+
+### Set up your MFA
+
+You have to set up [Multi-Factor Authentication (MFA)][MFA].
+
+1. [Sign in to AWS GDS account](https://gds-users.signin.aws.amazon.com/console)
+2. Select or go to IAM service.
+3. Click on "Users" in the menu bar on the left hand side
+4. Enter your name
+5. Click on the link for your email address
+6. Click on the security credentials tab
+7. Click on the "Manage" link next to "Assigned MFA device"
+8. Follow the steps to set up your MFA device
+
+> If you have a GDS-issued Yubikey, follow the [cross-GDS Yubikey docs](https://re-team-manual.cloudapps.digital/yubikeys.html#use-yubikey-for-2fa-in-amazon-web-services).
+
+### Generate a pair of access keys
+
+You have to generate an AWS Access Key and Secret Key to be able to
+perform operations with AWS on the command-line.
+
+1. [Sign in to the gds-users AWS Console](https://gds-users.signin.aws.amazon.com/console).
+1. Click on your email address in the top right.
+1. Click 'My Security Credentials'.
+1. Click 'Create access key'.
+1. Copy/paste them into the inputs that the gds-cli provides for you,
+   if you're following [the setup instructions](#first-run).
+
+#### Changing your MFA device
+
+1. Follow steps 1 to 7 in [set up your MFA][#set-up-your-mfa]
```

Comment:
Should be `[set up your MFA](#set-up-your-mfa)`

Reply:
Oops, thanks!